lib/get.js

```diff
@@ -0,0 +1,15 @@
+/**
+* `get` the value of a prop inside an object
+* if value is undefined return default
+*
+* @param {Objects} object to inspect
+* @param {String} property name
```

Comment:
Should we have get take a path? like `'a.b.c'` or should we separate to 2 functions? `prop` and `path`. Should the default value be optional? What do you think @danswater

Reply:
right now I'm thinking `path` should be a simple property name, if we we're going to take a path like this `'a.b.c'` then we should also consider this `a[0].b.c`. And yeah you're right default value should be optional I'll update the test case.

Reply:
@ktilcu just let me know if you guys would like path to support this scenario e.g `'a.b.c'`, `'a[0].b.c'` or `'a.b[0].c'` etc... and I will update the test case for that scenario.